lib/dependencies/CssImportDependency.js

```diff
@@ -17,19 +17,23 @@ const ModuleDependency = require("./ModuleDependency");
 /** @typedef {import("../ModuleGraph")} ModuleGraph */
 /** @typedef {import("../ModuleGraphConnection")} ModuleGraphConnection */
 /** @typedef {import("../ModuleGraphConnection").ConnectionState} ConnectionState */
+/** @typedef {import("../serialization/ObjectMiddleware").ObjectDeserializerContext} ObjectDeserializerContext */
+/** @typedef {import("../serialization/ObjectMiddleware").ObjectSerializerContext} ObjectSerializerContext */
 /** @typedef {import("../util/Hash")} Hash */
 /** @typedef {import("../util/runtime").RuntimeSpec} RuntimeSpec */
 
 class CssImportDependency extends ModuleDependency {
 	/**
 	 * @param {string} request request
 	 * @param {[number, number]} range range of the argument
+	 * @param {string | undefined} layer layer
```

Comment:
Can you describe the layer means here. Just a bit more description would be helpful.

Reply:
It is `@import url("landscape.css") layer(forms) screen and (orientation: landscape);`, so it is not related to `layer` feature inside webpack :smile: who works with CSS knowns about this feature, but yeah I can put an `@import` example here